pkg/osbuild/oscap_remediation_stage.go

```diff
@@ -28,6 +28,7 @@ type OscapConfig struct {
 	HtmlReport   string              `json:"html_report,omitempty" toml:"html_report,omitempty"`
 	VerboseLog   string              `json:"verbose_log,omitempty" toml:"verbose_log,omitempty"`
 	VerboseLevel OscapVerbosityLevel `json:"verbose_level,omitempty" toml:"verbose_level,omitempty"`
+	Compression  bool                `json:"compress_results,omitempty" toml:"compress_results,omitempty"`
```

Comment:
To clarify the above comment, these stage options are passed to the tailoring stage too... which is probably not great since some of these fields will break the tailoring stage if they're added somehow. I'm inclined to separate these.

Reply:
Agree with your inclination :)
Let's separate them.